tests/ui-fulldeps/stable-mir/crate-info.rs

```diff
@@ -39,7 +39,7 @@ fn test_stable_mir(_tcx: TyCtxt<'_>) -> ControlFlow<()> {
     assert!(get_item(&items, (DefKind::Fn, "foo::bar")).is_some());
 
     // Find the `std` crate.
-    assert!(stable_mir::find_crate("std").is_some());
+    assert!(stable_mir::find_crates("std").get(0).is_some());
```

Comment:
Maybe assert that it only finds one crate -- just in case there's something wrong with the filtering logic or sth?

Reply:
yeah makes sense